### DISCUSSION TITLE
data format get changed

### DISCUSSION THREAD

**Discussion by WoWoJiaJia2025** [Koushikphy/Interactive_Data_Editor#55] [Q&A]
Hi Koushik,

Your tool is amazing. Thank you for providing the software. I just have one problem with using it. After I edited my data interactively, if I saved the file, the data format will be changed. I wonder if there is a way to preserve my original data format. Basically, I just deleted several points from my file. 

before:
  507081583.000  0.3497931658717E-06  0.9202552064824E-06  0.5704620406107E-06 00000000
after:
5.0708158e+8	3.4979317e-7	9.2025521e-7	5.7046204e-7	0.0000000

Thanks.

**Comment by Koushikphy** [comment]
When saving, the software saves the data as rounded to 8 significant digits with exponent notation. It does so to save very large and very small numbers at the same time using a single format style. That is why those number gets rounded off to 8 significant digits. I have plans to provide an option where users can give different format styles for different columns of the dataset. Maybe that will be helpful for your work.

P.S. - It's nice to hear this software was useful for your work.
